source/adapters/hip/device.cpp

```diff
@@ -891,6 +891,13 @@ UR_APIEXPORT ur_result_t UR_APICALL urDeviceGetInfo(ur_device_handle_t hDevice,
     return ReturnValue(ur_bool_t{false});
   case UR_DEVICE_INFO_USM_POOL_SUPPORT:
     return ReturnValue(ur_bool_t{true});
+  case UR_DEVICE_INFO_BFLOAT16:
```

Comment:
I'm curious, since this sticks out to me: should `BFLOAT16`s not ideally have capabilities, like other floating-point formats?

Also, would bfloat16 support on HIP not depend on some device-specific features?

Reply:
`UR_DEVICE_INFO_BFLOAT16` I think should have been removed after https://github.com/intel/llvm/pull/13351 but think maybe it was lost in the UR changes when the PI was removed.

bfloat16 is supported on all backends on all devices via software implementations as a minimmum.

Reply:
I can make an issue to remove it, that isn't a change for this PR.

Reply:
https://github.com/oneapi-src/unified-runtime/issues/2385